src/allotropy/parsers/mabtech_apex/mabtech_apex_structure.py

```diff
@@ -50,19 +60,34 @@ def _create_measurement(plate_data: SeriesData) -> Measurement:
         sample_identifier=f"{well_plate}_{location_id}",
         detection_type=constants.DETECTION_TYPE,
         device_type=constants.DEVICE_TYPE,
-        exposure_duration_setting=plate_data.get(float, "Exposure"),
-        illumination_setting=plate_data.get(float, "Preset Intensity"),
+        exposure_duration_setting=plate_data.get(float, exposure_duration_setting_key),
+        illumination_setting=plate_data.get(float, illumination_setting_key),
         processed_data=ProcessedData(
             identifier=random_uuid_str(),
             features=[
-                ImageFeature(
-                    identifier=random_uuid_str(),
-                    feature=feature,
-                    result=plate_data.get(float, feature, NaN),
-                )
+                _build_feature(feature, plate_data, led_filter)
                 for feature in constants.IMAGE_FEATURES
             ],
         ),
+        custom_information_document=CustomInformationDocument(
+            led_filter=plate_data.get(str, "LED Filter") or NOT_APPLICABLE
+        ),
+    )
+
+
+def _build_feature(
```

Comment:
Can you instead make this `_format_feature_name` and just return the feature string (leaving the `ImageFeature` logic above) this makes the function scope as small and clear as possible.